pkg/reconciler/autoscaling/reconciler.go

```diff
@@ -175,19 +175,21 @@ func (c *Base) ReconcileMetric(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	desiredMetric := resources.MakeMetric(ctx, pa, metricSN, config.FromContext(ctx).Autoscaler)
 	metric, err := c.Metrics.Get(ctx, desiredMetric.Namespace, desiredMetric.Name)
 	if errors.IsNotFound(err) {
-		metric, err = c.Metrics.Create(ctx, desiredMetric)
+		_, err = c.Metrics.Create(ctx, desiredMetric)
```

Comment:
Why this change?

Reply:
Because the variable is used nowere after moving the else block in.